src/main/resources/reference.conf

```diff
@@ -1345,12 +1348,15 @@ resourceTypes = {
       custodian = {
         roleActions = ["manage_schema", "create_datasnapshot", "ingest_data", "soft_delete", "hard_delete", "unlink_snapshot", "list_snapshots", "lock_resource", "unlock_resource", "share_policy::custodian", "share_policy::ingester", "share_policy::snapshot_creator"]
         includedRoles = ["snapshot_creator"]
+        descendantRoles = {
```

Comment:
It wasn't obvious to me how this works but I'm hoping that `descendantRoles` is shared along with `roleActions` when `includedRoles` is used.

Reply:
`included_roles` is like role inheritance. The `custodian` includes everything from the `snapshot_creator` role. `descendantRoles` are those that are applied to children, grandchildren, etc. The `steward` role is granted to to all descendant `datasnapshot`s.